### DISCUSSION TITLE
Feature Request: MFA for members that are part of projects.

### DISCUSSION THREAD

**Discussion by vivekayer** [umbraco/Umbraco.Cloud.Issues#913] [Ideas]
### Issue description

Currently you are able to control and enforce MFA for members that are only part of the organisation.

![image](https://github.com/user-attachments/assets/33d19db8-0560-40c0-a34f-def6a9094103)

It will be good to be able to manage and enforce MFA for members at a project level.

**Comment by FransdeJong** [comment]
I second this request.
We are running into more and more clients that request MFA to be required and there is no way for us to enforce this on a project level.

It also feels like a big oversight in this day and age where security is top of mind.

**Comment by mttblss** [comment]
Exactly what Frans has said.

Every time I ask about this the reply is "You can enforce MFA", but that is missing the point, because it is only at the organisation level and is not suitable for Umbraco Partners providing cloud services to clients

- The agency is the organisation, 
- The clients have the projects
- MFA cannot be enforced for projects
- The clients require MFA for their projects to meet modern security standards

To have a B2B client project where MFA is opt-in by each user is not suitable
